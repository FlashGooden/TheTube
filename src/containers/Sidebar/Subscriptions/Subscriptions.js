import React, { Component } from 'react'
import {Subscription} from './Subscription/Subscription'
import {Divider} from 'semantic-ui-react'
import {SideBarHeader} from '../SideBarHeader/SideBarHeader'

export class Subscriptions extends Component {
  render() {
    return (
      <>
      <SideBarHeader title='subscriptons'/>
      </>
    )
  }
}
