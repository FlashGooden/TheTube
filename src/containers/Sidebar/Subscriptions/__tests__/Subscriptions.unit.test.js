import React from 'react'
import {shallow} from 'enzyme'
import Subscriptions from '../Subscriptions'
import {Menu} from 'semantic-ui-react'

test('should render Subscriptions', ()=> {
  const wrapper = shallow(
    <Menu>
      <Subscriptions />
    </Menu>
    
  )
  expect(wrapper).toMatchSnapshot()
})