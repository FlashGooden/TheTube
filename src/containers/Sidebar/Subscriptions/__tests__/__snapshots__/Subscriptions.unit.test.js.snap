// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Subscriptions 1`] = `
<div
  className="ui menu"
>
  <Component />
</div>
`;
